evmcore/tx_pool.go

```diff
@@ -212,9 +212,9 @@ func (config *TxPoolConfig) sanitize() TxPoolConfig {
 		log.Warn("Sanitizing invalid txpool journal time", "provided", conf.Rejournal, "updated", time.Second)
 		conf.Rejournal = time.Second
 	}
-	if conf.PriceLimit < 1 {
-		log.Warn("Sanitizing invalid txpool price limit", "provided", conf.PriceLimit, "updated", DefaultTxPoolConfig.PriceLimit)
-		conf.PriceLimit = DefaultTxPoolConfig.PriceLimit
+	if conf.MinimumTip < 1 {
```

Comment:
nit: is a minimum tip of 1 unit really needed?

Reply:
That is a good question, probably not, but it depends on the policies the networks wants to enforce. That said, its analysis should be part of a follow-up.